multinet/uploaders/swagger/nested_json.yaml

```diff
@@ -0,0 +1,29 @@
+Upload a nested JSON file to a collection of tables
+---
+consumes:
+  - text/plain
+
+parameters:
+  - $ref: "#/parameters/workspace"
+  - $ref: "#/parameters/table"
+  - name: data
+    in: body
+    description: Raw JSON content of file
+    schema:
+      type: object
```

Comment:
Can we add a richer example here?

Reply:
See https://github.com/multinet-app/multinet/pull/205#discussion_r354479906.

Reply:
50daf7efb8268f5f7a3970dcf620e9fec59df838

Reply:
8be5a8589f1f75f486416e91b8f484ff3c0be2f0